sql/moz-fx-data-shared-prod/fenix_derived/meta_attribution_country_counts_v1/query.sql

```diff
@@ -0,0 +1,28 @@
+-- Aggregate meta attribution counts by country and channel.
+SELECT
+  DATE(submission_timestamp) AS date_attributed,
+
```

Comment:
You need to run the linter on these files: `bqetl format sql/moz-fx-data-shared-prod/fenix_derived/meta_attribution_country_counts_v1`